airflow/utils.py

```diff
@@ -617,11 +619,20 @@ def handle_timeout(self, signum, frame):
         raise AirflowTaskTimeout(self.error_message)
 
     def __enter__(self):
-        signal.signal(signal.SIGALRM, self.handle_timeout)
-        signal.alarm(self.seconds)
+        try:
+            signal.signal(signal.SIGALRM, self.handle_timeout)
+            signal.alarm(self.seconds)
+        except Exception as e:
```

Comment:
Yeah I usually `logging.exception(e)`, but since timeouts are not expected to work when using `LocalExecutor` I thought this might throw people off. Some of these logging messages can really pollute your stdout while say running a backfill if they show 32 times.